torch/_dynamo/variables/tensor.py

```diff
@@ -121,19 +121,6 @@ def as_proxy(self):
     def python_type(self):
         return self.class_type
 
-    def call_isinstance(self, tensor_type):
-        def check_type(ty):
-            if ty not in tensortype_to_dtype:
-                return issubclass(self.python_type(), ty)
-
-            dtypes = tensortype_to_dtype[ty]
-            return self.dtype in dtypes
-
-        if type(tensor_type) is tuple:
-            return any(check_type(ty) for ty in tensor_type)
-        else:
-            return check_type(tensor_type)
-
```

Comment:
@voz did you remove this?  If so why?

Reply:
It's moved into `torch/_dynamo/variables/builtin.py` - this is the only place that has this pattern, and it was causing a bug, where we were trying to invoke this incorrectly.